src/Streamly/External/ByteString.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Streamly.External.ByteString
+  ( fromArray
```

Comment:
Arrange the operations in construction, elimination, transformation order like in Streamly.Memory.Array.